### ISSUE TITLE
Question w.r.t SEblock.

### ISSUE THREAD

**Issue by Gomdoree** [autonomousvision/projected_gan#30]
Hello.

First, thanks for this very cool project.

I have a question w.r.t SEblock.

Do you have any specific reason why bias option is set to "False" in conv layer of SEblock?

I know that the FastGAN and MobileNet also have the same option.

Thank you.

**Comment by xl-sr** [comment]
hey :)

The generator uses batch normalization in every UpBlock, hence, you can drop the bias term here. 
In the SE-block specifically, there is no BatchNorm, so in theory, you could add the bias back. However, this would not really improve the effective modeling capacity much, so you can simply save the compute.
